bat/BLDDIST.BAT

```diff
@@ -0,0 +1,77 @@
+@if /I "%1" == "tron" (echo on) else (echo off)
+rem Script to create z390 distribution zip file
+
+setlocal
+if /I "%1" == "tron" (set z_TraceMode=tron
+                      shift /1
+              ) else (if /I "%1" == "troff" (set z_TraceMode=troff
+                                             shift /1
+                                     ) else (set z_TraceMode=)
+                      )
+set /A z_NestLevel=%z_NestLevel%+1
+pushd %~dps0..
```

Comment:
between lines 11 and 12 please insert
rem ----- Lvl(%z_NestLevel%) Start %0 %1 %2 %3 %4 %5 %6 %7 %8 %9

Reply:
Added in 956070c